device/rsvp/rsvpInterface.yaml

```diff
@@ -0,0 +1,81 @@
+components:
+  schemas:
+    Rsvp.Interface:
+      x-status: under-review
+      description: >-
+        Configuration for RSVP Interface.
+      type: object
+      required: [ipv4_name,neighbor_ip]
+      properties:
+        ipv4_name: 
+          description: >-
+            The globally unique name of the IPv4 interface connected to the DUT.            
+          type: string
+          x-constraint:
+          - '/components/schemas/Device.Ipv4/properties/name'
+        neighbor_ip:
+          description: >-
+            IPv4 address of the RSVP neighbor on this interface.
+          type: string
+          format: ipv4
+        label_space_start:
+          description: >-
+            The user-defined label space start value. The LSPs for which this router acts as a egress are assigned labels from this
+            label pool. 
+          type: integer
+          default: 1000
+          minimum: 0
+          maximum: 1048575
+        label_space_end:
+          description: >-
+            The user-defined label space end value.
```

Comment:
label space end value is obvious from the name. I think we need to be more descriptive. Something like  "The last (user-defined) label value that can be used in RSVP message"